python/cuml/testing/strategies.py

```diff
@@ -521,3 +531,92 @@ def standard_regression_datasets(
     strategies.
     """
 )
+
+
+@composite
+def standard_classification_datasets(
```

Comment:
Possibly a follow-on, and I'm not even sure if I like this idea myself, but how would we feel about replacing a few randomly selected rows in these datasets with arrays generated by Hypothesis? That would keep the overall dataset mostly sensible but would test our robustness against more unusual inputs as well.

Reply:
Instead of doing that, I would suggest to take the approach used for `standard_regression_datasets` vs. `regression_datasets`. The "standard" versions are somewhat normal datasets produced with the "standard" functions, whereas the non-qualified strategies draw from both those and more arbitrary hypothesis arrays. See: https://github.com/rapidsai/cuml/blob/e9636fb8781e152b94b69dddf5db54412f07b22c/python/cuml/testing/strategies.py#L524-L533

Reply:
Love it. Let's do that as a follow-on.

Reply:
Recorded in https://github.com/rapidsai/cuml/issues/4960 .